doc/content/integrations/cloud-integrations/telemetry2u/index.md

```diff
@@ -0,0 +1,45 @@
+1. [Activate a 14-day free trial](https://telemetry2u.com/Identity/Account/Manage/Subscriptions) or create a PAYG plan with Telemetry2U.
+1. Devices already added to The Things Stack under an application.
+
+## Add Device to Telemetry2U
+First, [login to your Telemetry2U account](https://telemetry2u.com/Identity/Account/Login) and navigate to __Node Maintenance__ via __Admin__ from the top menu.
+
+Click the __Create New__ link in the top right corner to add a new LoRaWAN device.
+
+Complete the form as required.  The __Description__, __Profile__, __Device EUI__, __Network Provider__ and __Device Configuration__ are the minimum requirements. If your device is currently unsupported leave __Device Configuration__ at the default value.  Click the __Create__ button when complete.
+
+![Add a device in Telemetry2U](add-device.png "Add a device in Telemetry2U")
+
+## Setup The Things Stack Integration
+From your Telemetry2U account, navigate to __Setup Integration__ via __Admin__ from the top menu. Select __The Things Network__ from the __Network Provider__ drop down box and hit the __Add Integration__ button.
+
+![Add an integration](add-integration.png "Add an integration")
+
+Select the __Region__ your TTN account is registered under and make a copy of the __Telemetry2U Token__ with the __Copy to clipboard__ button.
+
+Retrieve your __Application ID__ from The Things Stack and enter it into the __Application ID__ field.
+
+The __API Key__ won't be known until the TTN integration has been created so click the __Save__ button and leave the browser tab open.
+
+![Setup the integration](setup-integration.png "Setup the integration")
+
+Log into The Things Stack and open your application.  Using the side menu, navigate to __Webhooks__ via __Integrations__ and click the __Add webhook__ button in the top right corner.
+
+![TTN Webhook](ttn-webhook.png "TTN Webhook")
+
+Choose __Telemetry2U__ as the webhook template.
+
+Use __telemetry2u__ as the __Webhook ID__ and enter the __Telemetry2U Token__ you copied to the clipboard earlier. This will be an alphanumeric string approximately 100 characters long.
+
+Use The Things Stack side menu to navigate to __API Keys__, click the downlink API key and copy the __Key ID__ from there into the __API Key__ section back on the integration configuration page on Telemetry2U. Click the __Save__ button to complete the integration.
+
+![TTN API](ttn-api.png "TTN API")
+
+__Congratulations__ your integration is now complete and once data has been received by TTN it should become available under the [Reports Page](https://telemetry2u.com/Reports). 
+
+If you're using an unsupported device you can select __Raw Data__ as the report type to confirm data is being received. You may then forward a copy of the device documentation including the packet format to Telemetry2U support and we'll get device support added ASAP. Please also include the EUI of your device in the e-mail.
```

Comment:
```suggestion
{{< note >}} If you're using an unsupported device on Telemetry2U, you can select **Raw Data** as the report type to confirm data is being received. You may then forward a copy of the device documentation including the payload format and device EUI to Telemetry2U support and they will get your device's support added ASAP. {{</ note >}}
```

Reply:
can you please add a link to support page on "Telemetry2U support" phrase in this sentence? or if it's a support email, use this format to link it "[ email ] ( mailto : support @ example.com )" (without blanks)